sdk/search/search-documents/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Release History
 
+## 12.1.0-beta.2 (2024-05-07)
+
+### Breaking Changes
+
+- Fixed an incorrect enum variant in `KnownVectorQueryKind` [#29601](https://github.com/Azure/azure-sdk-for-js/pull/29601)
```

Comment:
nit: wouldn't this technically be a bugfix?

Reply:
The value was correct, but the name of the variant in customer code is incorrect. If someone were to be using that enum, their code would fail to compile. Do you think marking that as a bugfix would be more clear to customers?

Reply:
If previous name doesn't work I'd treat it as a bug fix.

Reply:
The previous name does work. Customers can have `KnownVectorQueryKind.$DO_NOT_NORMALIZE$_text` in code that builds and runs correctly today.

Reply:
Then I think it's fine to list here.

Reply:
No strong feelings here, I think there's valid reasons why it could be listed as a bugfix and valid reasons why it could be listed as a breaking change. Especially between betas, I'm happy we added an entry regardless of where it sits😄